### ISSUE TITLE
RC2 - Device stats freeze after failed exploit, "try again" always fails

### ISSUE THREAD

**Issue by nsasso23** [Sticktron/g0blin#268]
Title

**Comment by andrewathalye** [comment]
Try reboot, aeroplane mode, reinstall g0nlin, wait for <5% usage. Maybe Also try to reinstall cydia

**Comment by DirkT86** [comment]
Like the post says above wait until the device is running around 3-5% on the very left hand meter in the g0blin app!  Has worked nearly every time for my flawlessly!  Even Resprings itself when it's done

**Comment by yuyi1005** [comment]
Me too, on mini4 10.3.2. 
"Try again" never works. And after failed exploit, the system is unstable until reboot.

**Comment by xhj14012** [comment]
Try again just means to reboot from now(latest):D

**Comment by Basputaek13** [comment]
Jailbreak

**Comment by nsasso23** [comment]
everything andrew's mentioned is already been done each time (save reinstall goblin, done that a few times). Haven't tried reinstalling cydia as I'm not sure how that'd help but i'll look into how to do that

**Comment by andrewathalye** [comment]
Try again always fails still? What device and ios?

**Comment by nsasso23** [comment]
iPhone SE 10.3.1 RC2

**Comment by andrewathalye** [comment]
Hmm. That should work. And you've tried deleting g0blin and reinstalling the ipa? Do you have backups?

**Comment by nsasso23** [comment]
Backups of what exactly? And just want to make clear the device still gets jailbroken but if the exploit fails it always reboots on the next try, and I keep trying after until it works.

**Comment by andrewathalye** [comment]
Does it ever work? If so, try downloading cydia eraser and then rejailbreaking - if you have a backup of your phone.

**Comment by nsasso23** [comment]
Yeah it works, but on the likely chance it doesn't, it will always reboot instead of succeeding. I'll try Cydia eraser, do you think that may also fix my other issue I just posted about? #274

**Comment by andrewathalye** [comment]
I don't know, but that sounds like a reinstall might fix it.

**Comment by nsasso23** [comment]
cydia eraser will not open. What can I do to fix?

**Comment by andrewathalye** [comment]
Hmm. I dunno at that point. I guess your phone is kind of toasted until g0blin r3...maybe try installing cydia eraser under meridian jailbreak. You can't install cydia, but you should be able to install cydia eraser.

**Comment by YoungSavage23** [comment]
The suggestions in this thread helped. Airplane mode and waited. Worked first try when i did that.

**Comment by HLixir** [comment]
i didn't even check if mine was under 5% but when i used my vpn it started to work. I'm not eve sure if that would help
